drivers/doc.txt

```diff
@@ -43,3 +43,10 @@
  * @ingroup     drivers
  * @brief       Drivers for storage devices
  */
+
+/**
+ * @defgroup    drivers_can CAN Drivers
+ * @ingroup     drivers
+ * @ingroup     can
```

Comment:
TBH I don't even know, why it has its own API... Looking at the API, all of that could have been done with `netdev`...